lib/screens/v2/candidate_generator/widgets/reconstructed_alert.ex

```diff
@@ -386,4 +388,12 @@ defmodule Screens.V2.CandidateGenerator.Widgets.ReconstructedAlert do
       true -> :severe
     end
   end
+
+  defp showing_departures?(%Screen{
+         app_params: %PreFare{departures: %Departures{sections: [_ | _]}, template: :duo}
+       }),
+       do: true
+
+  defp showing_departures?(_),
+    do: false
```

Comment:
This logic is exactly the same as in the candidate generator `pre_fare.ex`. First thought is that this should be consolidated/shared in case it changes (i.e. not just duos in the future), but I'm not sure the best place for that

Reply:
Not sure if `screens/util.ex` might be a good place to put it? Otherwise I could even create a separate util under `screens/v2/` somewhere?